components/eam/bld/config_files/definition.xml

```diff
@@ -63,9 +63,9 @@ Switch to turn on/off advecting CLUBB moments: 0 => no, 1 => yes
 PBL package: uw (University of Washington), hb (Holtslag and Boville), hbr
  (Holtslag, Boville, and Rasch), clubb_sgs.
 </entry>
-<entry id="rad" valid_values="rrtmg,rrtmgp,camrt" value="">
+<entry id="rad" valid_values="rrtmg,rrtmgp" value="">
 Radiative transfer calculation: 
-camrt (CAM3 and CAM4 RT package), rrtmg (RRTMG package from AER), rrtmgp (RRTMGP package from AER).
+rrtmg (RRTMG package from AER), rrtmgp (RRTMGP package from AER).
```

Comment:
Should we bother defining what "AER" stands for here?

Reply:
I can link to the source websites if you want. AER is Atmospheric and Environmental Research, the group behind both projects. I guess this isn't relevant now that we don't have CAMRT to distinguish between different sources for these codes. I could also refer to the relevant papers here, but this is far more detail than anything else in this file.

Reply:
I just meant to define the acronym to make it easier for someone to look it up themselves.